modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -2669,6 +2669,86 @@ TEST_P(Test_ONNX_layers, YOLOv7)
     normAssertDetections(refClassIds, refScores, refBoxes, keep_classIds, keep_confidences, keep_boxes);
 }
 
+
+TEST_P(Test_ONNX_layers, YOLOv5n)
```

Comment:
Wrong fixture. Use `Test_ONNX_nets`: `TEST_P(Test_ONNX_nets, YOLOv5n)`

Reply:
Please also fix for `TEST_P(Test_ONNX_layers, YOLOv7)` -> `TEST_P(Test_ONNX_nets, YOLOv7)`